src/backend/templates/rooms.hbs

```diff
@@ -851,6 +851,19 @@
       function loadVideoAndSlides(div, videoURL, slideURL){
         if(videoURL !== null && videoURL !== '') {
           let isVideoDisplayed = $('#desc-' + div).hasClass('in');
+
+          if (videoURL.indexOf('v=') !== -1 && videoURL.indexOf('&feature=youtu.be') === -1) {
+            videoURL = videoURL.split('v=')[1];
+          }
+          else if (videoURL.indexOf('v=') !== -1 && videoURL.indexOf('&feature=youtu.be') !== -1) {
+            const endIndex = videoURL.indexOf('&feature=youtu.be');
+            const startIndex = videoURL.indexOf('v=');
+            videoURL = videoURL.substring(startIndex+2, endIndex);
+          }
```

Comment:
You could save a few lines, if you were willing to use a regexp to strip any unwanted param or hash.
```js
if (videoUrl.indexOf('v=') !== -1) {
  videoUrl = videoUrl.split('v=')[1].replace(/[&#].*/, '');
}
```
Pros and cons! :wink:

(I was also wondering if `new URL(videoUrl).searchParams.get('v')` might work, but that's [not yet](https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams) available in IE or Edge.)

Reply:
Thanks for the `regex` solution, I will make the changes. :smile: